src/app/flask_app.py

```diff
@@ -59,6 +59,12 @@ def process_metadata():
     restrict_to_themes = request.args.get("Restrict to Themes")
     if restrict_to_themes:
         restrict_to_themes = restrict_to_themes.split(",")
+    
+    exclude_deprecated = request.args.get("excludeDeprecated", "false").lower() == "true"
+
+    #exclude_deprecated = True
+    # Print the value of exclude_deprecated to the console
+    print(f"exclude_deprecated: {exclude_deprecated}")
```

Comment:
These lines seem to just be for debugging, I'd remove them

Reply:
Ah yes! forgot to remove them, cheers